### ISSUE TITLE
auto answer doesnt work

### ISSUE THREAD

**Issue by Misiekes** [LiterallyFabian/auto-starquiz#1]
it showing answers in console but i need to click manually

**Comment by salmonslay** [comment]
Yes, that is correct. This script is for solving the questions, not answering them. See the "Contribution" header in the project readme file.
